### PR TITLE
feat: HOTEL-13433 cancel penalty description

### PR DESCRIPTION
**Pull request by shubhadasc**
Updated docs to support following changes.
Added new cancel penalty description field to CancelPenalties Field inside CancelPenalty is changed from required to optional.

### CODE REVIEW

**Review by sachin-raja — APPROVED** [review]

**Comment by RoAnnC1** [comment]
@shubhadasc The email you used to create the commit  is not associated with your GitHub account. Make sure to add (and verify) your email to your GitHub account.

Afterwards you can trigger a recheck via [https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=](https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=946)

**Comment by shubhadasc** [comment]
> @shubhadasc The email you used to create the commit is not associated with your GitHub account. Make sure to add (and verify) your email to your GitHub account.
> 
> Afterwards you can trigger a recheck via [https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=](https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=946)

@RoAnnC1 Can you please review and merge the changes. Thank you!

File: src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md [hunk 1]
```diff
@@ -1772,6 +1777,7 @@ Details about the total pricing associated with stay.
 
 |Name|Type|Format|Description|
 |---|---|---|---|
+`description`|`string`|-|-|
```
[the remark this answers is not in the document]
Comment (shubhadasc) [reply src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md]: Done!

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**